ckanext/who_afro/templates/group/read.html

```diff
@@ -30,6 +30,7 @@ <h2> {{ _('Search data') }}</h2>
     <form {% if form_id %}id="{{ form_id }}" {% endif %}class="search-form{% if no_bottom_border %} no-bottom-border{% endif %}"  method="get" data-module="select-switch">
         <div class="input-group">
           <input aria-label="{% block header_site_search_label %}{{ placeholder }}{% endblock %}" id="field-giant-search" type="text" class="form-control input-lg" name="q" value="{{ query }}" autocomplete="off" placeholder="{{ placeholder }}">
+          <input type="hidden" name="groups" value="indicators">
```

Comment:
But this means that if we ever try and search on a different page... it will always only return indicators?

Reply:
(In the future the insights page will probably be a group page)

Reply:
I see what you mean. Will find a way to add some logic such that different groups are differentiated.

Reply:
@ChasNelson1990 The last commit should address the concern you just raised. Thank you!
(Also, I just rebased with `master` and got new changes that don't belong here but hopefully can be ignored.)